shared/message/platform/webots/messages.proto

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+
+syntax = "proto3";
+
+package message.platform.webots;
+
+// Copy of cyberbotics/webots projects/samples/contests/robocup/controllers/player/messages.proto
+
+message Vector3 {
+    double X = 1;
+    double Y = 2;
+    double Z = 3;
+}
+
+message PositionSensorMeasurement {
+    string name  = 1;
+    double value = 2;
+}
+
+message AccelerometerMeasurement {
+    string  name  = 1;
+    Vector3 value = 2;  // [m/sˆ2], x-axis, y-axis, z-axis
+}
+
+message GyroMeasurement {
+    string  name  = 1;
+    Vector3 value = 2;  // [rad/s], x-axis, y-axis, z-axis
+}
+
+message BumperMeasurement {
+    string name  = 1;
+    bool   value = 2;
+}
+
+message ForceMeasurement {
+    string name  = 1;
+    double value = 2;  // [N]
+}
+
+message Force3DMeasurement {
+    string  name  = 1;
+    Vector3 value = 2;  // [N], x-axis, y-axis, z-axis
+}
+
+message Force6DMeasurement {  // not yet implemented
```

Comment:
What do you mean "not yet implemented"? Are we waiting on us, or the TC or something?

Reply:
That'll be Cybertronics.

Reply:
We don't really use it anyway.